### DISCUSSION TITLE
The request body does not seem to contain the information of the consumer connector.

### DISCUSSION THREAD

**Discussion by TP9-dev** [eclipse-edc/Samples#243] [Q&A]
In the Samples/transfer/transfer-00-prerequisites/README.md, when registering the data plane instance for the consumer connector, only the port numbers like 19192 and 19291, which appear to be associated with the provider connector, are registered in the request body. Information about the consumer connector is not included. Can you explain why this is?

**Comment by ndr-brt** [comment]
in a transfer interaction there's no need to register the data-plane on the consumer side, and indeed in that readme it's not listed
